core/src/main/java/org/frankframework/monitoring/Trigger.java

```diff
@@ -45,8 +45,9 @@
 public class Trigger implements ITrigger {
 	protected Logger log = LogUtil.getLogger(this);
 
-	private static final String CLASS_NAME_ALARM = Alarm.class.getName();
-	private static final String CLASS_NAME_CLEARING = Clearing.class.getName();
+	// The element names, which can be used as a Trigger.
+	private static final String ALARM_NAME = "AlarmTrigger";
+	private static final String CLEARING_NAME = "ClearingTrigger";
```

Comment:
Eeeuurw 😢

Reply:
Het is niet de mooiste oplossing, maar het alternatief is ook niet goed. De Frank!Doc ziet de class `Alarm` met als superclass `Trigger` en knoopt die twee aan elkaar. We kunnen de naam van beide classes pakken en ze bij elkaar voegen, maar dat is ook niet echt future proof imo.

Reply:
Echt stom dat de doc nu de interfacenaam er standaard achter plakt maargoed...

Reply:
Kunnen we niet een `@ff.defaultElement` op Alarm zetten?

Reply:
Ik vind dit er ook niet zo sjiek uitzien. Ik mis geloof ik wat dit met de frank doc te maken heeft. Waarom kunnen we niet obv de class de XML opmaken in #toXml?

Reply:
`FRANKDOC.md` documents that `@ff.defaultElement` has the following behaviour: "Set default value of className attribute in XSD syntax 1 element.". This is nice for old syntax elements, such as pipe, but does not change the name of the element. I don't think that there is another way to change the name of the element.